### ISSUE TITLE
empty page for unused address

### ISSUE THREAD

**Issue by knarz** [wmitsuda/otterscan#67]
When navigating to an address that hasn't been used before (.e.g. `/address/0x73a77B7719DEc6678f383075898737f77859775c`, you are presented with an empty page and the javascript console spits out the following error:

```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'hash')
    at oe (AddressTransactions.tsx:267)
    at oa (react-dom.production.min.js:157)
    at Va (react-dom.production.min.js:180)
    at ja (react-dom.production.min.js:178)
    at Gu (react-dom.production.min.js:274)
    at Is (react-dom.production.min.js:250)
    at Os (react-dom.production.min.js:250)
    at Cs (react-dom.production.min.js:250)
    at gs (react-dom.production.min.js:243)
    at react-dom.production.min.js:123
```

**Comment by wmitsuda** [comment]
for some reason this issue got out of my radar, sorry, I'll take a look at it soon!

**Comment by wmitsuda** [comment]
Fixed by #67, it should be in next release.
